shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/screens/AboutAppScreen.kt

```diff
@@ -98,6 +107,12 @@ fun AboutAppScreen(
                     }
                     .padding(16.dp)
             )
+
+            Image(
+                imageVector = vectorResource(Res.drawable.made_with_junie),
+                contentDescription = stringResource(Res.string.about_app_made_with_junie),
+                modifier = Modifier.align(Alignment.CenterHorizontally)
+            )
```

Comment:
Let's make it clickable and open landing page

Reply:
![image](https://github.com/user-attachments/assets/18376b20-285e-4d21-a80d-22cbde041fac)